### PR TITLE
TLPORTAL-237: Add vagrant configuration and rake file tasks

### PR DESCRIPTION
**Pull request by dlhaines**
To make it possible to do simple and consistent tomcat testing add a vagrant configuration and corresponding Rakefile tasks to run Student Dashboard in a standalone VM.

### CODE REVIEW

File: vagrant/vagrantXterm.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+xterm -e 'vagrant ssh' &
```
Comment (lsloan) [inline vagrant/vagrantXterm.sh]: Why an xterm?  Why not something more general?  xterm isn't available everywhere.  Specifying xterm for OS X is kind of like specifying Apple's Terminal.app for Linux.

File: vagrant/ARTIFACTS.TXT [hunk 2]
```diff
@@ -0,0 +1 @@
+$ARTIFACTS_SRC=/Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/StudentDashboard
```
Comment (lsloan) [inline vagrant/ARTIFACTS.TXT]: This file doesn't appear to be used.  Delete it?

File: vagrant/VERSIONS.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Contains variable definitions for the StudentDashboard Vagrant install.
+# Location of build artifacts on host server.
+ARTIFACTS_SRC=/Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/StudentDashboard
```
Comment (lsloan) [inline vagrant/VERSIONS.sh]: Is it necessary to use an absolute path that includes your username?  Why not use a relative path based on the "vagrant" directory?

File: vagrant/vagrantXterm.sh (hunk 1 above)
Comment (dlhaines) [inline vagrant/vagrantXterm.sh]: It's meant as a developer shortcut.  I'll see if term works.

File: vagrant/ARTIFACTS.TXT (hunk 2 above)
Comment (dlhaines) [inline vagrant/ARTIFACTS.TXT]: Good catch.

File: vagrant/VERSIONS.sh (hunk 3 above)
Comment (dlhaines) [inline vagrant/VERSIONS.sh]: I'll improve the explanation and keep that line, but just as an example.

File: vagrant/vagrantXterm.sh (hunk 1 above)
Comment (dlhaines) [inline vagrant/vagrantXterm.sh]: xterm is the most general utility I can find.  I'm happy to consider a pull request if you can find something better.

Comment (lsloan) [inline vagrant/vagrantXterm.sh]: Do a check:  If xterm exists, use it.  If Terminal.app exists, use it.  Otherwise, just open ssh in the current shell.

**Comment by bkirschn** [comment]
looks good

File: vagrant/vagrantXterm.sh (hunk 1 above)
Comment (dlhaines) [inline vagrant/vagrantXterm.sh]: Will extend code (or consider a pull request) if this surfaces as an issue in practice.

**Pull request merged**